model/metrics/rpc-metrics.yaml

```diff
@@ -13,8 +13,7 @@ groups:
       - ref: network.type
       - ref: server.address
       - ref: server.port
-      - ref: server.socket.address
-      - ref: server.socket.port
+      - ref: server.ip
```

Comment:
it's not clear to me whether we need a replacement for `server.socket.port`?

Reply:
I believe it translates into `server.port` or `proxy.port` depending on who's the peer, which should be enough